homeassistant/components/binary_sensor/openuv.py

```diff
@@ -24,30 +25,28 @@
 ATTR_PROTECTION_WINDOW_ENDING_UV = 'end_uv'
 
 
-async def async_setup_platform(
```

Comment:
You probably want to keep `async_setup_platform` around but just do nothing, because people might have already configured it.